.travis.yml

```diff
@@ -300,6 +300,11 @@ matrix:
       env: SWIGLANG=ruby VER=2.6
       sudo: required
       dist: xenial
+    - compiler: gcc
+      os: linux
+      env: SWIGLANG=ruby VER=2.7.0
+      sudo: required
+      dist: xenial
```

Comment:
I've added in ruby-2.7 testing on Travis to master in 5259082cb175737ffc1712587571927711855773. Can you please remove the changes to .travis.yml and push an updated patch? Then we can then see how your patch performs using 2.7.0p0 on Travis. Thanks.

Reply:
Actually Travis ran its tests just fine with my newly committed change (where 2.7.0 instead of 2.7 is requested), see https://travis-ci.org/swig/swig/jobs/631045861?utm_medium=notification&utm_source=github_status .

Reply:
Nice one, I don't know how you found that was supported as `rvm list known` doesn't include 2.7.0p0, just 2.7[.0-preview1]. Anyway, either approach tests what we want which is good.

Reply:
Just a gut feeling that there might be some fallback for fully specified version strings. However the final patch does not include that one in favor of your more elaborate solution and to resolve the conflicts.